### PR TITLE
redis - fill out docs and update format

### PR DESCRIPTION
**Pull request by technovangelist**

### CODE REVIEW

File: content/integrations/redis.md [hunk 1]
```diff
@@ -1,36 +1,84 @@
...
+## Configuration Options
+
+* `unix_socket_path` - (Optional) - Can be used instead of `host` and `port`.
+* `db`, `password`, and `socket_timeout` - (Optional) - Additional connection options.
+* `warn_on_missing_keys` - (Optional) - Display a warning in the info page if the keys we're tracking are missing.
+* `slowlog-max-len` - (Optional) - Maximum number of entries to fetch from the slow query log. By default, the check will
+        read this value from the redis config. If it's above 128, it will default to 128 due to potential increased latency
+        to retrieve more than 128 slowlog entries every 15 seconds. If you need to get more entries from the slow query logs
+        set the value here. Warning: It may impact the performance of your redis instance
+* `command_stats` - (Optional) - Collect INFO COMMANDSTATS output as metrics.
```
Comment (irabinovitch) [inline content/integrations/redis.md]: Generally like this level of detail.  Should we confider this in other integrations as well?

File: content/integrations/redis.md [hunk 2]
```diff
@@ -1,36 +1,84 @@
...
+        ======
+
+          [...]
+
+          redisdb
+          -------
+            - instance #0 [OK]
+            - Collected 8 metrics & 0 events
+
+#  Troubleshooting and Questions
```
Comment (irabinovitch) [inline content/integrations/redis.md]: I dont think we have this section for other integration?

Should we link to the redis guide? https://www.datadoghq.com/blog/monitor-redis-using-datadog/

File: content/integrations/redis.md (hunk 1 above)
Comment (technovangelist) [inline content/integrations/redis.md]: Been trying to get in the habit for last few. Generally moving this content from the comments in the yaml into this options subsection.

File: content/integrations/redis.md (hunk 2 above)
Comment (technovangelist) [inline content/integrations/redis.md]: Troubleshooting is a standard heading in the read me that I added a few months back. Rarely use it unless there is something already there

**Pull request merged**